presto-tests/src/test/java/io/prestosql/tests/TestInformationSchemaConnector.java

```diff
@@ -107,7 +107,7 @@ public void testLargeData()
         assertQuery("SELECT count(*) from test_catalog.information_schema.tables WHERE table_schema = 'test_schema1'", "VALUES 100000");
         assertQuery("SELECT count(*) from test_catalog.information_schema.tables WHERE table_name = 'test_table1'", "VALUES 2");
         assertQuery("SELECT count(*) from test_catalog.information_schema.columns WHERE table_schema = 'test_schema1' AND table_name = 'test_table1'", "VALUES 100");
-        assertEquals(METADATA_CALLS_COUNTER.get() - metadataCallsCountBeforeTests, 12);
+        assertEquals(METADATA_CALLS_COUNTER.get() - metadataCallsCountBeforeTests, 13);
```

Comment:
I see increase in call count. This is bad.

Reply:
fixed